docs/common/registration.md

```diff
@@ -2,17 +2,207 @@ Registering in the OSG
 ======================
 
 The OSG keeps a registry containing active projects, virtual organizations (VOs), resources, and resource
-downtimes in the [topology Github repository](https://github.com/opensciencegrid/topology/).
+downtimes stored as as [YAML files](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html)
+in the [topology GitHub repository](https://github.com/opensciencegrid/topology/).
 This registry is used for [accounting data](https://gracc.opensciencegrid.org), contact information, and resource
 availability, particularly if your site is part of the [World LHC Computing Grid](http://wlcg.web.cern.ch/) (WLCG).
 
 Use this page to learn how to register information in the OSG.
 
+Registration Requirements
+-------------------------
+
+The instructions in this document require the following:
+
+- A [GitHub](https://github.com/) account
+- A working knowledge of [GitHub collaboration](https://help.github.com/categories/collaborating-with-issues-and-pull-requests/)
+- OSG contact registration. If not already registered [here](https://topology.opensciencegrid.org/miscuser/xml?), send
+  an email to <mailto:help@opensciencegrid.org> with the following information:
+    - Full name
+    - GitHub ID
+    - Description of your OSG affiliation, e.g. FermiGrid site administrator, senior scientist for the DUNE experiment,
+      etc.
+    - Contact information of site, Virtual Organization, or project sponsor to prove affiliation
+
+Registering Resources
+---------------------
+
+An OSG resource is a host that provides grid services, e.g. Compute Elements, storage endpoints, or perfSonar hosts.
+See the full list of services that should be registered in the OSG topology
+[here](https://github.com/opensciencegrid/topology/blob/master/topology/services.yaml).
+
+OSG resources are stored under a hierarchy of facilities, sites, and resource groups, defined as follows:
+
+| Level          | Definition                                                                                                                                                             |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Facility       | The institution or company where your resource is located, e.g. `University of Wisconsin`                                                                              |
+| Site           | Smaller than a facility; typically represents an academic department, research group, or a computing center, e.g. `CHTC` for the Center for High Throughput Computing. |
+| Resource Group | A logical grouping of resources, e.g. you may group together resources that serve your Slurm cluster under a resource group named `Slurm-HPC`                          |
```

Comment:
We want the resource group to be globally unique so tell people to put the site name in the RG, too, e.g. `CHTC-Slurm-HPC`.